pick.lic

```diff
@@ -85,7 +85,10 @@ class Pick
     @disarm_careful_threshold = @settings.pick['disarm_careful_threshold'] || 5
     @disarm_too_hard_threshold = @settings.pick['disarm_too_hard_threshold'] || 10
     @too_hard_container = @settings.pick['too_hard_container']
+	@autoloot_container = @settings.autoloot_container
```

Comment:
🧹  This appears to maybe be a TAB character, for formatting consistency, would you please replace them with spaces? Thanks!

https://www.youtube.com/watch?v=SsoOG6ZeyUI